### ISSUE TITLE
Can't change postcss.config file name

### ISSUE THREAD

**Issue by Yankovsky** [michael-ciniawsky/postcss-load-config#127]
I use postcss-loader and I want to specify path to my postcss config file.
```
{
  loader: postcssLoader,
  options: {
    config: {
      path: path.resolve(__dirname, '../../ui-kit.postcss.config.js')
    }
  }
}
```
Notice, that my file name is not postcss.config.js, but ui-kit.postcss.config.js. With that config I get error:
```
Module build failed: ModuleBuildError: Module build failed: Error: No PostCSS Config found in: /Users/yankovskyandrey/Code/hr/packages/csssr-ui-kit/ui-kit.postcss.config.js
```
If I change the name of file to postcss.config.js, than everything is ok.

I dove in source code and found that problem is in integration between postcss-loader, postcss-load-config and cosmiconfig package. 
postcss-loader calls postcss-load-config `postcssrc(rc.ctx, rc.path, { argv: false })`, then postcss-load-config calls cosmicconfig `config('postcss', options)`. Inside cosmiconfig:
```
module.exports = function (moduleName, options) {
  options = assign({
    packageProp: moduleName,
    rc: '.' + moduleName + 'rc',
    js: moduleName + '.config.js',
    argv: 'config',
    rcStrictJson: false,
    stopDir: oshomedir(),
    cache: true,
  }, options);
```
and then 
```
        var directory = (searchPathIsDirectory)
          ? absoluteSearchPath
          : path.dirname(absoluteSearchPath);
        return searchDirectory(directory);
```
and in searchDirectory
```
return loadJs(path.join(directory, options.js));
```

So, basically, file name is ignored.

**Comment by michael-ciniawsky** [comment]
> Notice, that my file name is not postcss.config.js, but ui-kit.postcss.config.js. 

This is intended and by design, `cosmiconfig` supports a few config naming conventions with a given base name (`postcss`). You can put the configs in separate folders `ui-kit.postcs.config.js` => `.config/ui-kit/postcss.config.js`. Custom file names will never get direct support from `postcss-load-config`, since this renders the conventional aspect basically useless (which was one of the main goal behind this project)

**Comment by Yankovsky** [comment]
Yeah, I used this workaround, but still from package user point of view this moment is unclear.
Maybe if it is intended, than it is better to change postcss-loader parameter from `config.path` to `config.directory` or add some warning in documentation.
